api/processes.json

```diff
@@ -1,7 +1,7 @@
 {
   "apps" : [{
     "name"               : "mockiji-api",
-    "script"             : "app.js",
+    "script"             : "app.js --config-file ../../mocks/data/dev.json",
```

Comment:
This path does not exists in this standalone context. 
I would like people to be able to clone this repository and launch it without editing anything and they would be able to use the "/mocks/api-simple-library" as described in the "Get started with default configuration" part from the README.

Also, I think it's important to be able to override this default configuration from an untracked file.
Before you're merge-request, I was on the idea of creating a env.json.dist file and use it to create a env.json file if this last file didn't exist.